distribution/std-lib/Base/src/System/File.enso

```diff
@@ -106,6 +107,13 @@ type Input_Stream
     close : Unit
     close = Managed_Resource.finalize this.stream_resource
 
+    ## Exposes operations on the underlying Java input stream.
+
+       Useful when integrating with polyglot functions requiring an
+       `InputStream` as an argument.
+    with_java_stream : (Java_Input_Stream -> Any) -> Any
```

Comment:
The name doesn't imply `InputStream`. Use `with_java_input_stream`?

Reply:
This is on an object called `Input_Stream`, so I'd say the fact this is an input stream is kinda given.

Reply:
Ah, I missed that, sorry.